geomstats/geometry/symmetric_matrices.py

```diff
@@ -42,7 +59,18 @@ def get_basis(self):
     @staticmethod
     @geomstats.vectorization.decorator(['matrix'])
     def to_vector(mat):
-        """Convert the symmetric part of a symmetric matrix into a vector."""
+        """Convert the symmetric part of a symmetric matrix into a vector.
```

Comment:
Redudant: convert the symmetric matrix or the symmetric part of a matrix? As is_symmetric is checked, I suggest the former

Reply:
Done.